### ISSUE TITLE
SOLIDLY contract fee mismatch declared on UI

### ISSUE THREAD

**Issue by omelched** [solidlyexchange/solidly.exchange#10]
Hi!
I was browsing through SOLIDLY LPs’ contract code, specifically [sAMM-SOLIDsex/SOLID](https://ftmscan.com/address/0x62e2819dd417f3b430b6fa5fd34a49a377a02ac8#code) when noticed some kind of mismatch.

As i understood, those lines in swap() method gather fees:

```
if (amount0In > 0) _update0(amount0In / 10000); // accrue fees for token0 and move them out of pool
if (amount1In > 0) _update1(amount1In / 10000); // accrue fees for token1 and move them out of pool
```

and seems like they gather 1/10000 = **0.01%** from token

but on [SOLIDLY website home page](https://solidly.exchange/home) there is a banner which states **«0.001% fee»**

Could you please specify whether I’m wrong (and where I'm wrong) or there indeed is a slight UI issue?

**Comment by antonnell** [comment]
Issue with the UI. It's being updated in the next patch.
![image](https://user-images.githubusercontent.com/38530524/155879739-4202fde6-c455-49a2-91c1-31f091119c93.png)

**Comment by antonnell** [comment]
Fixed with 68b39fea5dbe84fe6e80437630bec936525bdacc
![image](https://user-images.githubusercontent.com/38530524/155884583-a45b0e28-0225-4e35-a934-1f043c76c20b.png)
